internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go

```diff
@@ -0,0 +1,318 @@
+package cluster_mgrs
```

Comment:
For when the PR is refined, can we add debugging logging similarly to scale up mgr?

Reply:
+1, i'll add more log lines on this. 

I was thinking of metrics too for the two reconcilers (I am open to suggestions on them).

Reply:
added the debug lines.